libraries/SD/src/SD.h

```diff
@@ -136,7 +136,7 @@ class SDClass {
         uint64_t sz = size64();
 #ifdef DEBUG_ESP_PORT
 	if (sz > (uint64_t)SIZE_MAX) {
-            DEBUG_ESP_PORT.printf_P(PSTR("WARNING: SD card size overflow (%lld>= 4GB).  Please update source to use size64().\n"), sz);
+            DEBUG_ESP_PORT.printf_P(PSTR("WARNING: SD card size overflow (%lld>= 4GB).  Please update source to use size64().\n"), (long long)sz);
```

Comment:
`std::numeric_limits<uint32_t>::max()` as rhs?
plus host `(u64)val > (u64)(SIZE_MAX)` never works since `SIZE_MAX` is u64 already

Reply:
Good catch, cleaned up!